.travis.yml

```diff
@@ -129,6 +137,10 @@ jobs:
         - target/cli/linux/conduit install --conduit-version=$CONDUIT_TAG |tee conduit.yml
         - kubectl -n conduit apply -f conduit.yml --prune --selector='conduit.io/control-plane-component'
 
+  allow_failures:
+    - rust: nightly
```

Comment:
let's also add `fast_finish: true` so build status can be reported before this job finishes.